library/user.inc

```diff
@@ -136,3 +136,49 @@ function prevSetting($uspfx, $postvar, $label, $initval)
 
     return $pset;
 }
+
+function collectAndOrganizeExpandSetting($current_filename, $filenames = array(), $global_value = 0)
```

Comment:
Recommend removing the 3rd parameter since you can just get the global directly inside the function. This then makes the function calls less awkward. For example, in the typical case where there is on a filename (and not a group), you just need to call the function with 1 parameter.

Reply:
`$global_value = $GLOBALS['expand_form']);` inside the `function collectAndOrganizeExpandSetting`
does not work.  That was why I passed it as an argument.